h/views/activity.py

```diff
@@ -319,7 +319,8 @@ def toggle_tag_facet(request):
         tag_facets = _tag_facets(request, parsed_query)
         tag_facets.remove(tag)
         if tag_facets:
-            parsed_query['tag'] = tag_facets
+            for tag_facet in tag_facets:
+                parsed_query['tag'] = tag_facet
```

Comment:
Yeah, your're right. This whole thing should be rewritten. This works:
```
        tag_facets = _tag_facets(request, parsed_query)
        tag_facets.remove(tag)
        del parsed_query['tag']
        if tag_facets:
            for tag_facet in tag_facets:
                parsed_query.add('tag', tag_facet)

```

I'll also change the test to have 3 tags